ibis/__init__.py

```diff
@@ -53,6 +53,10 @@
     # pip install ibis-framework[mapd]
     import ibis.mapd.api as mapd  # noqa: F401
 
+with suppress(ImportError):
+    # pip install ibis-framework[spark]
```

Comment:
This will need corresponding changes in `setup.py` if you haven't already made them.